### ISSUE TITLE
[Artwork] Price isn't shown on thumbnails in grid views

### ISSUE THREAD

**Issue by jorystiefel** [artsy/eigen#750]
See screenshot. We include the artist name and artwork title, but no the price.
![ios simulator screen shot aug 24 2015 3 06 14 pm](https://cloud.githubusercontent.com/assets/1065248/9449619/b75ed076-4a71-11e5-934f-cf79202b01da.png)

We'll need to add a price label to the `ARArtworkThumbnailMetadataView` class.

From `ARArtworkThumbnailMetadataView`:
```objc
- (void)configureWithArtwork:(Artwork *)artwork
{
    self.primaryLabel.text = artwork.artist.name;
    [self.secondaryLabel setTitle:artwork.title date:artwork.date];
}
```

**Comment by jorystiefel** [comment]
It would also be nice to properly truncate these labels (with an ellipsis).

**Comment by alloy** [comment]
Is truncating preferred over expanding the label into multiple lines?

**Comment by jorystiefel** [comment]
@alloy There were some bigger architectural changes here. The grid layout uses several class methods for sizing that return fixed values, not easily allowing for variable heights for the metadata (2 lines or 3 lines).

**Comment by alloy** [comment]
Fixed by #763.
